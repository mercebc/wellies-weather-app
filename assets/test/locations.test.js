import * as React from 'react';
import * as ReactDOM from 'react-dom'

import Locations from '../js/locations';
const ReactTestRenderer = require('react-test-renderer');

test('renders the input field with placeholder', () => {
  const testInstance = ReactTestRenderer.create(
    <Locations />
  ).root;

  expect(testInstance.findByType('input').props.placeholder).toBe("Search...");
});
