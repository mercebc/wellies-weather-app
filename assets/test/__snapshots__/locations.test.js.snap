// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the input field 1`] = `
<div
  class="flex"
>
  <input
    class="margin-small"
    id="city"
    placeholder="Search..."
    type="text"
    value=""
  />
</div>
`;
